### DISCUSSION TITLE
Options for notifications via email per repo and daily digest, etc...

### DISCUSSION THREAD

**Discussion by 94rain** [community/community#7151] [Other Feature Feedback, Questions, & Ideas]
**Current issue:**
I set up both email and web notifications for my account for repos that I am watching. Some I'd like to get email notifications. Others have too many activities per day and I only want to get web notifications. But we are only able to change notifications options of email and/or web for the **whole user account**. 
![image](https://user-images.githubusercontent.com/21193371/139901104-dc2c6bfb-f1e1-43fc-a2fe-81bb71b62a9c.png)

**Potential solution:**

It'd be better if we can set the email/web notification options repo by repo, or opt-out of email notifications **for some specific repos** that have too many messages per day, or have a **daily digest / n hour per digest** option like the Dependabot alerts.

**Comment by karlcow** [comment]
It would be indeed very useful to be able to subscribe to a repo and set on which email you would like to receive notifications. Currently the only solution for this is to be part of the organization, but it's not really a practical solutions for all repos.
